### PR TITLE
Improve MegaMenu - Ready For Review

### PR DESCRIPTION
**Pull request by Seroxdesign**
## Overview

**What features/fixes does this PR include?**

This improves the mega-menu to showcase more links to users without having to expand the menus

**Please provide the GitHub issue number**

https://linear.app/metagame/issue/MG-17/clean-up-the-mega-menu-by-merging-or-removing-more-stuff

Closes #

## Follow up Improvement Ideas

- [ ] please list any improvement/ideas

## Implementation

**Describe technical (nontrivial / non-obvious) parts of your code**

**Side effects**

## Assets

[Include screenshots/videos if it makes reviewing easier.]

### CODE REVIEW

**Comment by Seroxdesign** [comment]
This is ready for review @alalonde

**Comment by Seroxdesign** [comment]
TODO: Switch the breakpoints to 1600 for desktop

File: packages/design-system/src/theme/index.ts
Comment (HHH-GH) [inline packages/design-system/src/theme/index.ts:1]: I was going to comment on this in the Discord, but was too late, sorry.

Changing the value for the breakpoints here is going to affect other things in the site that were using them for the layouts. I know I did some work on designs that depended on the `xl` breakpoint being 80em.

If the changes to the breakpoints are to work only for the MegaMenu, would it work to write some custom css just for that component?

Comment (Seroxdesign) [reply packages/design-system/src/theme/index.ts:1]: Changing breakpoints was recommended by @dysbulic, I believe I tested a few pages out here so I'm not sure we will have any breaking issues. 

Although i did as much changes to actual component as I could without going out of the design provided to me.

Open to changing it if I **need** to but would rather not

Comment (alalonde) [reply packages/design-system/src/theme/index.ts:1]: I agree, we probably don't want to change the breakpoints for the entire site. Can you just override them for the MegaMenu ?

Comment (Seroxdesign) [reply packages/design-system/src/theme/index.ts:1]: I'll look into it

Comment (Seroxdesign) [reply packages/design-system/src/theme/index.ts:1]: Okaya returned breakpoints and fixed it in the components, can you guys take a look and review now @HHH-GH @alalonde

Comment (HHH-GH) [reply packages/design-system/src/theme/index.ts:1]: I'd be happy to compare the design with what we're seeing in the deploy, but I think I need a Linear.app invite. 

for e.g. it looks like the MG logo has dropped out, but not sure if that's on purpose.

Here's the current site 👇

image


On this deploy I'm not seeing that logo 👇

image

Comment (Seroxdesign) [reply packages/design-system/src/theme/index.ts:1]: Good catch, I'll get that fixed soon

Comment (Seroxdesign) [reply packages/design-system/src/theme/index.ts:1]: logo is back, check with Peth in builders-guild for access to linear

**Review by alalonde — APPROVED** [review]

**Pull request merged**